sources/frontend.c

```diff
@@ -881,7 +881,9 @@ od_frontend_cleanup(od_client_t *client, char *context,
 		break;
 
 	default:
```

Comment:
I think that using abort is more appropriate here, because this condition could only happen in case of programming error and not related to the protocol.

Reply:
Yup, I have 6 core dumps.
But they are useless: there is not enough information.
Appropriate logging would be much better.

Reply:
I've removed handling of corrupted statuses with attempt of logging. @pmwkaa can you plz look into this again?

Reply:
I agree that it would be better to fix the root of problem, but it will be easier to find roots of problems with logging. So LGTM

Reply:
Finally, that was "unexpected error status OD_DETACH (4)". But how could we find this out without logs?